src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -122,32 +122,32 @@ private void testActions() {
 
         ______TS("Typical case: no picture");
 
-        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Usual Nationality",
+        profilePage.editProfileThroughUi("", "short.name", "e@email.tmt", "inst", "Singaporean",
                                          "male", "this is enough!$%&*</>");
-        profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Usual Nationality",
+        profilePage.ensureProfileContains("short.name", "e@email.tmt", "inst", "Singaporean",
                                           "male", "this is enough!$%&*</>");
         profilePage.verifyStatus(Const.StatusMessages.STUDENT_PROFILE_EDITED);
-        
+
```

Comment:
> Could you please point me to the code that handled this case before the fix?

Therein lies exactly the problem: since previous method was using textbox input, there is no "invalid input" per se and thus there is no handling. Now that the nationality input is restricted to an enumerable set, many of the existing data in the live server will consist of invalid nationality, which should be handled.

> Could you please tell me what do you mean by properly handled?

Let's suppose that a student has previously entered his/her nationality as "Singaprean" (a typo). Under your new code, "Singaprean" is not an acceptable nationality, but the data already exists in the live server. What would the dropdown input show? Is it "Other" or the default "--- Select ---" or others? It is up to you to decide, I cannot prescribe any default behaviour; I will let it pass as long as it is reasonable.